### PR TITLE
fix(chat): fix chat name generating for link attachments (Issues #1273, #1274)

### PR DESCRIPTION
**Pull request by Alexander-Kezik**
**Description:**

Fix chat name generating for link attachments

Issues:

- #1273 
- #1274

**Checklist:**

- [x] the pull request name complies with [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/)
- [x] the pull request name starts with `fix(<scope>):`, `feat(<scope>):`, `feature(<scope>):`, `chore(<scope>):`, `hotfix(<scope>):` or `e2e(<scope>):`. If contains breaking changes then the pull request name must start with `fix(<scope>)!:`, `feat(<scope>)!:`, `feature(<scope>)!:`, `chore(<scope>)!:`, `hotfix(<scope>)!:` or `e2e(<scope>)!:` where `<scope>` is name of affected project: `chat`, `chat-e2e`, `overlay`, `shared`, `sandbox-overlay`, etc.
- [x] the pull request name ends with `(Issue #<TICKET_ID>)` (comma-separated list of issues)
- [x] I confirm that do not share any confidential information like API keys or any other secrets and private URLs

### CODE REVIEW

**Comment by Alexander-Kezik** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-1304.nightly-test.deltixhub.io
>E2E tests status: success

File: apps/chat/src/utils/app/conversation.ts [hunk 1]
```diff
@@ -93,16 +93,21 @@ export const getNewConversationName = (
...
+
+    if (!title && reference_url) {
+      return reference_url;
+    }
+
+    return title;
   }
 
-  return convName;
+  return conversation.name;
```
Comment (IlyaBondar) [inline apps/chat/src/utils/app/conversation.ts]: please, revert this change with `prepareEntityName(conversation.name);` and add `prepareEntityName(reference_url)`

File: apps/chat/src/store/conversations/conversations.epics.ts [hunk 2]
```diff
@@ -1112,10 +1113,11 @@ const sendMessageEpic: AppEpic = (action$, state$) =>
           assistantMessage,
         );
 
-        const newConversationName =
+        const newConversationName = prepareEntityName(
```
Comment (IlyaBondar) [inline apps/chat/src/store/conversations/conversations.epics.ts]: Please, revert changes into this file and see comment in `getNextDefaultName`

**Review by IlyaBondar — APPROVED** [review]
